tests/unit_tests/initialisation/test_aerosol_init.py

```diff
@@ -0,0 +1,53 @@
+# pylint: disable=missing-module-docstring
+import numpy as np
+import pytest
+from chempy import Substance
+
+# from PySDM.initialisation import spectra
+from PySDM.initialisation.aerosol_composition import DryAerosolMixture
+from PySDM.physics import si
+
+# spectrum = spectra.Lognormal(norm_factor=100.0 / si.cm**3, m_mode=50.0 * si.nm, s_geom=2.0)
+
+
+@pytest.mark.parametrize(
+    "mass_fractions",
+    [
+        pytest.param({"(NH4)2SO4": 1.0, "insoluble": 0.0}),
+        pytest.param({"(NH4)2SO4": 0.5, "insoluble": 0.5}),
+        pytest.param({"(NH4)2SO4": 0.0, "insoluble": 1.0}),
+    ],
+)
+def test_aerosol_init(mass_fractions):
```

Comment:
perhaps let's rename the test to something more descriptive like `test_volume_weighted_kappa_with_insoluble_compound()`?